doc/repoquery.rst

```diff
@@ -75,6 +75,9 @@ Together with ``<pkg-spec>``, control what packages are displayed in the output.
 ``--whatrequires <capability>``
     Limit the resulting set only to packages that require ``<capability>``.
 
+``--alldeps``
+    Expand the resulting set of --whatrequires to packages that also require files that ``<capability>`` provides.
```

Comment:
OK, looking at the Bugzilla and at the implementation, it seems I know what's going on now.
So, I think that it would be worth mentioning that it expands the result only if `<capability>` is a package name and thus that it expands it with those "packages that requires any of the capabilities provided by the packages named `<capability>`".